### PR TITLE
Review nutritional facts in Guides > Basics

### PR DESCRIPTION
**Pull request by christinechoy-okta**
## Description:
- **What's changed?** Review nutritional facts in Basics section of the Guides
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-456109](https://oktainc.atlassian.net/browse/OKTA-456109)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/quickstart/main/index.md [hunk 1]
```diff
@@ -3,27 +3,27 @@ title: QuickStart - Signing in your first user
 excerpt: Get started with Okta authentication
 ---
 
-In this article we get you up and running with basic Okta authentication, quickly and effectively. This includes creating and managing Okta organizations, registering a new app and authenticating it using Okta, and next steps from there.
-
 <StackSelector />
 
+This guide explains how to get you up and running with basic Okta authentication, quickly and effectively. This includes how to create and manage Okta orgs, register a new app, and authenticate the app by using Okta, and the subsequent steps from there.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/quickstart/main/index.md:8]:
```suggestion
This guide explains how to get you up and running with basic Okta authentication, quickly and effectively. This includes how to create and manage Okta orgs, register a new app, authenticate the app by using Okta, and the subsequent steps from there.
```

File: packages/@okta/vuepress-site/docs/guides/saml-tracer/main/index.md [hunk 2]
```diff
@@ -1,46 +1,46 @@
...
-This article explains how to test a SAML app implementation with the SAML Tracer browser add-on.
+This guide explains how to test a SAML app implementation with the SAML tracer browser add-on.
 
 ---
 
 **Learning outcomes**
 
 * Install and use SAML Tracer.
-* Create SAML requests for SP- and IdP-initiated flows and inspect them in SAML Tracer.
+* Create SAML requests for SP- and IdP-initiated flows, and inspect them in SAML tracer.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/saml-tracer/main/index.md:14]: This is compound sentence and doesn't require a comma before and
```suggestion
* Create SAML requests for SP- and IdP-initiated flows and inspect them in SAML tracer.
```

File: packages/@okta/vuepress-site/docs/guides/saml-tracer/main/index.md [hunk 3]
```diff
@@ -1,46 +1,46 @@
...
 
 **What you need**
 
-* A SAML app to test (see [Prepare a SAML integration](/docs/guides/build-sso-integration/saml2/main/#prepare-a-saml-integration) for more information on creating one).
+A SAML app to test. See [Prepare a SAML integration](/docs/guides/build-sso-integration/saml2/main/#prepare-a-saml-integration) for more information on how to create one.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/saml-tracer/main/index.md:18]:
```suggestion
A SAML app to test. See [Prepare a SAML integration](/docs/guides/build-sso-integration/saml2/main/#prepare-a-saml-integration).
```

File: packages/@okta/vuepress-site/docs/guides/saml-tracer/main/index.md [hunk 4]
```diff
@@ -1,46 +1,46 @@
...
 SAML tracer is an add-on in Firefox and very useful when troubleshooting SAML for Service Provider-initiated flows (SP-initiated)
 or Identity Provider-initiated flows (IdP-initiated).
 When you start an IdP-initiated flow or SP-initiated flow while SAML tracer is enabled, it captures the SAML request and response.
 
-## Install SAML tracer or Similar Browser Tool
+## Install SAML tracer or similar browser tool
 
 To install SAML tracer, visit [https://addons.mozilla.org/en-US/firefox/addon/saml-tracer/](https://addons.mozilla.org/en-US/firefox/addon/saml-tracer/) and follow the instructions.
 
 Once you install SAML tracer, open it from the browser menu bar: **Tools** > **SAML tracer**.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/saml-tracer/main/index.md:33]:
```suggestion
After you install SAML tracer, open it from the browser menu bar: **Tools** > **SAML tracer**.
```

File: packages/@okta/vuepress-site/docs/guides/saml-tracer/main/index.md [hunk 5]
```diff
@@ -1,46 +1,46 @@
...
 
-## SP-Initiated flow
+## SP-initiated flow
 
 To create a SAML request for an SP-initiated flow and inspect the request and response in SAML tracer:
 
-1. Open SAML tracer and then access your application, which takes you to the Okta login page if you aren't already logged in.
+1. Open SAML tracer and then access your application, which takes you to the Okta sign-in page if you aren't already logged in.
 2. Look at the SAML tracer window and see the SAML request sent from your application to Okta.
 3. Okta returns a SAML Response.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/saml-tracer/main/index.md:45]: This isn't a step. Can you move up to step 2 as the second sentence?

```suggestion
3. Okta returns a SAML Response.
```

File: packages/@okta/vuepress-site/docs/guides/sharing-cert/main/index.md [hunk 6]
```diff
@@ -4,26 +4,22 @@ excerpt: How to share application key credentials between Apps
...
 
-In this article you learn how to share application key credentials across multiple applications so that they can be authorized using the same Identity Provider.
+This guide explains how to share application key credentials across multiple applications so that they can be authorized by using the same Identity Provider.
 
 ---
 
 **Learning outcomes**
 
-* Create a new application key credential for your source app and update it to use the new certificate.
+* Create a new application key credential for your source app, and update it to use the new certificate.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/sharing-cert/main/index.md:13]:
```suggestion
* Create a new application key credential for your source app and update it to use the new certificate.
```

File: packages/@okta/vuepress-site/docs/guides/sign-your-own-saml-csr/main/index.md [hunk 7]
```diff
@@ -4,22 +4,22 @@ excerpt: How to use a custom SAML certificate for apps
...
 * Update your app and ISV to use the new certificate.
 * Clone the certificate for use across multiple apps.
 
 ---
 
-## Outbound and Inbound SAML Applications
+## Outbound and Inbound SAML apps
 
-Okta as a SAML Service Provider is referred to as the Inbound SAML. Okta as a SAML Identity Provider (IdP) is referred to as the Outbound SAML. The general procedure is the same for both. However, some of the API calls are different as described in the following sections. 
+Okta as a SAML Service Provider is referred to as the Inbound SAML. Okta as a SAML Identity Provider (IdP) is referred to as the Outbound SAML. The general procedure is the same for both. However, some of the API calls are different as described in the following sections.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/sign-your-own-saml-csr/main/index.md:22]:
```suggestion
Okta as a SAML Service Provider is referred to as Inbound SAML. Okta as a SAML Identity Provider (IdP) is referred to as Outbound SAML. The general procedure is the same for both. However, some of the API calls are different as described in the following sections.
```

File: packages/@okta/vuepress-site/docs/guides/updating-saml-cert/main/index.md [hunk 8]
```diff
@@ -1,41 +1,41 @@
...
 
 ---
 
 **Learning outcomes**
 
-* Find out which certificate your SAML app uses, and download your certificate
-* Use the [Apps API](/docs/reference/api/apps/#list-applications) to return data on the apps that need updating and generate new credentials for each app
-* Update the apps to use the new certificate
-* Learn how to revert to SHA1 if necessary
+* Find out which certificate your SAML app uses, and learn to download your certificate.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/updating-saml-cert/main/index.md:13]:
```suggestion
* Find out which certificate your SAML app uses and learn to download your certificate.
```

File: packages/@okta/vuepress-site/docs/guides/updating-saml-cert/main/index.md [hunk 9]
```diff
@@ -1,41 +1,41 @@
...
+* Use the [Apps API](/docs/reference/api/apps/#list-applications) to return data on the apps that need updating, and generate new credentials for each app.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/updating-saml-cert/main/index.md:14]:
```suggestion
* Use the [Apps API](/docs/reference/api/apps/#list-applications) to return data on the apps that need updating and generate new credentials for each app.
```

File: packages/@okta/vuepress-site/docs/guides/updating-saml-cert/main/index.md [hunk 10]
```diff
@@ -1,41 +1,41 @@
...
+* Update the apps to use the new certificate.
+* Learn how to revert to SHA1 if necessary.
 
 **What you need**
 
-* A SAML app to upgrade (see [Building a SAML SSO integration](/docs/guides/build-sso-integration/saml2/main/#prepare-a-saml-integration) for more information on creating one)
+* A SAML app to upgrade. See [Building a SAML SSO integration](/docs/guides/build-sso-integration/saml2/main/#prepare-a-saml-integration) for more information on creating one.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/guides/updating-saml-cert/main/index.md:20]:
```suggestion
* A SAML app to upgrade. See [Building a SAML SSO integration](/docs/guides/build-sso-integration/saml2/main/#prepare-a-saml-integration).
```

**Review by susanharper-okta — COMMENTED** [review]
couple things

File: packages/@okta/vuepress-site/docs/guides/saml-tracer/main/index.md (hunk 5 above)
Comment (christinechoy-okta) [reply packages/@okta/vuepress-site/docs/guides/saml-tracer/main/index.md:45]: Fixed 👍

**Review by susanharper-okta — APPROVED** [review]
LGTM 🚀

**Pull request merged**